app/helpers/application_helper/toolbar_chooser.rb

```diff
@@ -481,6 +481,7 @@ def center_toolbar_filename_classic
             resource_pool
             container_template
             ems_block_storage
+            storage_resource
```

Comment:
we shouldn't need empty toolbars if `storage_resource` is not added here

Reply:
removed